src/main/java/org/apache/mesos/offer/DefaultOfferRequirementProvider.java

```diff
@@ -12,22 +13,30 @@
  * A default implementation of the OfferRequirementProvider interface.
  */
 public class DefaultOfferRequirementProvider implements OfferRequirementProvider {
-    private final Logger logger = LoggerFactory.getLogger(getClass());
+    private static final Logger logger = LoggerFactory.getLogger(DefaultOfferRequirementProvider.class);
+
+    private final TaskConfigRouter configRouter;
```

Comment:
Ended up changing the constructor structure to allow passing in a custom TaskConfigRouter (now an interface) or to use a DefaultConfigRouter.